# DP: Description: Adding empty m68k framestate.c not to build for dwarf2.
# DP: Related bugs: 
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: GOTO Masanori <gotom@debian.org>
# DP: Upstream status: Debian-Specific
# DP: Status Details: m68k uses sjlj exceptions, not dwarf2 unwind.
# DP:		This patch adds empty framestate.c for m68k to avoid
# DP:		dwarf2 build failure.
# DP: Date: 2003-07-19 (Update 2005-03-16 gotom)

2005-03-16  GOTO Masanori  <gotom@debian.org>

	* sysdeps/m68k/unwind-pe.c: Update to fix compilation failure.

2003-07-19  GOTO Masanori  <gotom@debian.org>

	* sysdeps/m68k/framestate.c: Add to fix compilation failure
	with sjlj exception, suggested by Philip Blundell.


--- /dev/null   2003-01-13 18:16:28.000000000 +0900
+++ sysdeps/m68k/framestate.c    2003-07-16 23:21:34.000000000 +0900
@@ -0,0 +1,2 @@
+
+
--- /dev/null	2004-12-05 06:07:48.000000000 +0000
+++ sysdeps/m68k/unwind-pe.c	2005-03-14 05:14:16.000000000 +0000
@@ -0,0 +1,6 @@
+#include <stdlib.h>
+#include <unwind.h>
+
+#define _LIBC_DEFINITIONS
+#define NO_BASE_OF_ENCODED_VALUE
+#include "unwind-pe.h"
